src/transformers/models/electra/modeling_electra.py

```diff
@@ -338,6 +344,108 @@ def forward(
         return outputs
 
 
+# Copied from transformers.models.bert.modeling_bert.BertSdpaSelfAttention with Bert->Electra
+class ElectraSdpaSelfAttention(ElectraSelfAttention):
```

Comment:
hey! Would you mind updating the SDPA and non SDPA to also support flex attention? 
You have an example of this in `modeling_llama..py` with the `ATTENTION_INTERFACE`!